system-addon/content-src/components/Base/Base.jsx

```diff
@@ -49,6 +49,7 @@ class Base extends React.Component {
         <div className="outer-wrapper">
           <main>
             {prefs.showSearch && <Search />}
+            <ManualMigration />
```

Comment:
Looks like we should be able to conditionally show this like the adjacent components. Probably `!pref.migrationExpired` ?